chia/full_node/mempool_manager.py

```diff
@@ -590,10 +590,10 @@ async def new_peak(
             if last_npc_result.conds is not None:
                 for spend in last_npc_result.conds.spends:
                     if spend.coin_id in self.mempool.removals:
-                        c_ids: List[bytes32] = self.mempool.removals[bytes32(spend.coin_id)]
-                        self.mempool.remove_from_pool(c_ids)
-                        for c_id in c_ids:
-                            self.remove_seen(c_id)
+                        spendbundle_ids: List[bytes32] = self.mempool.removals[bytes32(spend.coin_id)]
```

Comment:
the name `removals` is also confusing then. That name usually refers to a list of coins that were removed. I guess in this case it's a `removed_coin` -> `SpendBundle` mapping

Reply:
Yep, it sure is. I'll put that in another PR!